TalZivelin/TZivelin-HW-W01.js

```diff
@@ -0,0 +1,125 @@
+1) (Difficulty: mixed) Predict the output of each of the following expressions, and explain the rules which determine it. If the output depends on the value of x, identify the conditions when the expression will output true. Assume the cases are independent, and x is reset to an unknown value before each.
+
+Some of these are tricky! Don't trust your first instinct.
+
+a) "1" == 1 - True because this does not check for the var types.
+
+b) "1" === 1 - Flase, the === checks that both type (string, NUmber) and value are identical.
+
+c) x == 'x' - If X is a number this will be true.
```

Comment:
1c) This will actually be false in almost all cases, though if `var x = 'x'` it will be true.